meta/signature/draft-1.json

```diff
@@ -1,56 +1,365 @@
+          "type": "string"
+        },
+        "Annotation": {
+          "description": "Annotation of the signature", 
+          "type": "string"
+        },
+        "ArrayExpress Accession": {
+          "description": "Accession number of the data on the ArrayExpress data store.", 
+          "type": "string"
+        },
         "Assay": {
-          "type": "string",
-          "description": "The assay (TODO: validate against official list)"
+          "description": "The assay (TODO: validate against official list)",
+          "type": "string"
         },
-        "Gene": {
-          "type": "string",
-          "description": "The gene (TODO: validate against official list)"
+        "Authors": {
+          "description": "The authors involved with the signature.", 
+          "type": "string"
+        },
+        "Batch": {
+          "description": "The batch", 
+          "type": "string"
+        },
+        "Biological Process": {
+          "description": "GO Biological Process.", 
+          "type": "string"
+        },
+        "Brain Region": {
+          "description": "Region of the brain where the samples were harvested", 
+          "type": "string"
+        },
+        "Breed": {
+          "description": "Breed of the sample", 
+          "type": "string"
+        },
+        "Category Code": {
+          "description": "Category code of the gene set.", 
+          "type": "string"
         },
         "Cell Line": {
-          "type": "object",
-          "properties": {
-            "Accession": {
-              "type": "string",
-              "description": "Accession number"
-            },
-            "Name": {
-              "type": "string",
-              "description": "Official name of cell line"
-            }
-          },
-          "required": [
-            "Name"
+          "properties": {"Accession": {
+                            "description": "Accession number",
+                            "type": "string"
+                          },
+                          "Name": {
+                            "description": "Official name of cell line",
+                            "type": "string"}
+                          },
+          "required": ["Name"],
+          "type": "object"},
+        "Cellular Component": {
+          "description": "GO Cellular Component", 
+          "type": "string"
+        },
+        "Cellular Compartment": {
+          "description": "Subcellular localization information", 
+          "type": "string"
+        },
+        "Chip": {
+          "description": "Source platform", 
+          "type": "string"
+        },
+        "Chromosome Location": {
+          "description": "Location in the chromosome", 
+          "type": "string"
+        },
+        "Complex": {
+          "description": "Protein complexes", 
+          "type": "string"
+        },
+        "Complex ID": {
+          "description": "Complex Identifier", 
```

Comment:
How is this different from the Complex? Might they both fit into `Complex: {name, id}`?